pkg/graveler/graveler.go

```diff
@@ -428,9 +428,9 @@ func (cp CommitParents) AsStringSlice() []string {
 const FirstCommitMsg = "Repository created"
 
 // CommitVersion used to track changes in Commit schema. Each version is change that a constant describes.
-type CommitVersion int
+type CommitVersion int32
```

Comment:
Why `int32`? usually it's `64` (platform defaults)

Reply:
We save this field in protobuf as int32 so there's no point anyway in keeping it int32